server/test/services/program/ProgramDefinitionTest.java

```diff
@@ -1426,7 +1432,10 @@ public void serializeThenDeserialize_allFieldsPresent() throws JsonProcessingExc
                 Locale.ITALIAN,
                 "Italian summary image description"));
     assertThat(result.externalLink()).isEqualTo("external.link");
-    assertThat(result.statusDefinitions().getStatuses())
+    assertThat(
+            applicationStatusesRepository
+                .lookupActiveStatusDefinitions(result.adminName())
+                .getStatuses())
```

Comment:
We can just remove this assert, and the concept of statuses from this test entirely, right?

No need to test the `ApplicationStatusesRepository` here

Reply:
In the next PR..